package.json

```diff
@@ -112,27 +112,32 @@
         "compile": "tsc -p ./ && node copyStaticFiles.js",
         "watch": "node copyStaticFiles.js && tsc -watch -p ./",
         "postinstall": "node ./node_modules/vscode/bin/install",
-        "test": "npm run compile && node ./node_modules/vscode/bin/test",
-        "unittest": "node_modules/.bin/mocha -u tdd out/unittest/*.js"
+        "pretest": "npm run compile",
+        "test": "node ./out/configure/test/runTest.js"
```

Comment:
can we have one command which builds as well as runs test. May be we can have two commands - `buildAndTest` and `test`

Reply:
When we push the code, checks are already there for checking build related issues. Therefore, I created a separate command which only runs the tests.